sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/Configuration.java

```diff
@@ -91,6 +91,9 @@
     @Parameter(names = "-manageDatabase", description = "Control switch for creating/deleting underlying database resource")
     private boolean manageDatabase = false;
 
+    @Parameter(names = "-preferredRegionsList", description = "Comma separated preferred regions list")
```

Comment:
this is a useful config, as we are adding a global config to the benchmark tool, could you please pass this config to the other components in the benchmark tool as well?

Reply:
I was debating it originally , now you mentioned that i added for all the workload. Thanks !